packages/suite-base/jest.config.json

```diff
@@ -24,5 +24,6 @@
     "@lichtblick/suite-base/(.*)": "<rootDir>/src/$1"
   },
   "//": "Native find is slow because it does not exclude files: https://github.com/facebook/jest/pull/11264#issuecomment-825377579",
-  "haste": { "forceNodeFilesystemAPI": true }
+  "haste": { "forceNodeFilesystemAPI": true },
+  "testEnvironment": "jsdom"
```

Comment:
This will not forcing all files to use jsdom in the tests environment? If yes, I would say to remove. It should be used only when the tests uses an API of the DOM, which is the case when we render components. But pure classes shouldn't use this, they should use the standard "node" environment. It is more efficient. Please verify it.